device/globals/esc_compile.gd

```diff
@@ -25,6 +25,7 @@ var commands = {
 	"teleport_pos": { "min_args": 3 },
 	"walk": { "min_args": 2 },
 	"walk_block": { "min_args": 2 },
+	"turn": { "min_args": 2 },
```

Comment:
I think it would be better to name the command (and the corresponding GDScript functions something like `set_angle`/`set_degree`, since `turn`, to me at least, implies that the item/player will always turn when it is called.

Reply:
This goes back to what @StraToN once said about preferring "humane" names. `set_angle` and `set_degree` are very explicit, but `turn` does always turn the item/player just as much as `set_angle` would.

Reply:
I'm not 100% convinced  this case is equivalent. `turn` doesn't turn the item/player if the player is already facing that direction. It sounds more like a toggle to me, while the others clearly define what will be achieved by running the command. I welcome @StraToN's input though.

Reply:
Or even worse, it might lead the developer to think they will turn the item/player by x degrees, which could be useful in some cases.

Reply:
But... `set_angle` doesn't do anything either if the player is already at that angle, unless the entire implementation is changed. And how would the player turn or have his angle set if he's already there?

If you (try to) open the can in the demo room, you can see that the player is also animated while turning, so there shouldn't be any visual confusion either.

If anything, `set_*` sounds more "toggly" to me.

Reply:
> Or even worse, it might lead the developer to think they will turn the item/player by x degrees, which could be useful in some cases.

The documentation says otherwise, and if we really find a need for that functionality, it'd be `turn_by` anyway.

Reply:
Who reads documentation though? :stuck_out_tongue: In that case, wouldn't it make sense to have the commands be `turn_to` and `turn_by`?